habitat-lab/habitat/tasks/nav/nav.py

```diff
@@ -446,35 +446,35 @@ def _get_observation_space(self, *args: Any, **kwargs: Any):
             dtype=np.float32,
         )
 
-    def get_agent_start_pos(self, episode, task):
+    def get_agent_start_position_and_rotation(self, episode, task):
```

Comment:
I would keep the API the same across this and Compass. Use `get_agent_start_position` and `get_agent_start_rotation`.